src/resume_parser/static/style.css

```diff
@@ -0,0 +1,38 @@
+
+body {
```

Comment:
Could you make the styling consistent with the styling of our web application so the colour and font are uniform with the rest of the app.

Reply:
Resolved in the next pull request